examples/baseline-app/docker-compose.yml

```diff
@@ -0,0 +1,631 @@
+    hostname: alice-redis
+    networks:
+      - alice
+    ports:
+      - 6379:6379
+    restart: always
+
+  alice-nats:
+    image: provide/nats-server
+    container_name: alice-provide-nats
+    command: ["-auth", "testtoken", "-p", "4222", "-D", "-V"]
+    environment:
+      JWT_SIGNER_PUBLIC_KEY: |-
+        -----BEGIN PUBLIC KEY-----
+        MIICIjANBgkqhkiG9w0BAQEFAAOCAg8AMIICCgKCAgEAullT/WoZnxecxKwQFlwE
+        9lpQrekSD+txCgtb9T3JvvX/YkZTYkerf0rssQtrwkBlDQtm2cB5mHlRt4lRDKQy
+        EA2qNJGM1Yu379abVObQ9ZXI2q7jTBZzL/Yl9AgUKlDIAXYFVfJ8XWVTi0l32Vsx
+        tJSd97hiRXO+RqQu5UEr3jJ5tL73iNLp5BitRBwa4KbDCbicWKfSH5hK5DM75EyM
+        R/SzR3oCLPFNLs+fyc7zH98S1atglbelkZsMk/mSIKJJl1fZFVCUxA+8CaPiKbpD
+        QLpzydqyrk/y275aSU/tFHidoewvtWorNyFWRnefoWOsJFlfq1crgMu2YHTMBVtU
+        SJ+4MS5D9fuk0queOqsVUgT7BVRSFHgDH7IpBZ8s9WRrpE6XOE+feTUyyWMjkVgn
+        gLm5RSbHpB8Wt/Wssy3VMPV3T5uojPvX+ITmf1utz0y41gU+iZ/YFKeNN8WysLxX
+        AP3Bbgo+zNLfpcrH1Y27WGBWPtHtzqiafhdfX6LQ3/zXXlNuruagjUohXaMltH+S
+        K8zK4j7n+BYl+7y1dzOQw4CadsDi5whgNcg2QUxuTlW+TQ5VBvdUl9wpTSygD88H
+        xH2b0OBcVjYsgRnQ9OZpQ+kIPaFhaWChnfEArCmhrOEgOnhfkr6YGDHFenfT3/RA
+        PUl1cxrvY7BHh4obNa6Bf8ECAwEAAQ==
+        -----END PUBLIC KEY-----
+    healthcheck:
+      test: ["CMD", "/usr/local/bin/await_tcp.sh", "localhost:4222"]
+      interval: 1m
+      timeout: 1s
+      retries: 2
+      start_period: 10s
+    hostname: alice-nats
+    networks:
+      - alice
+    ports:
+      - 4221:4221
+      - 4222:4222
+    restart: always
+    volumes:
+      - ./ops/await_tcp.sh:/usr/local/bin/await_tcp.sh:cached
+
+  alice-nats-streaming:
+    image: provide/nats-streaming
+    command: ["-cid", "provide", "--auth", "testtoken", "-SDV"]
+    container_name: alice-nats-streaming
+    depends_on:
+      - alice-nats
+    healthcheck:
+      test: ["CMD", "/usr/local/bin/await_tcp.sh", "localhost:4222"]
+      interval: 1m
+      timeout: 1s
+      retries: 2
+      start_period: 10s
+    hostname: alice-nats-streaming
+    networks:
+      - alice
+    restart: always
+    volumes:
+      - ./ops/await_tcp.sh:/usr/local/bin/await_tcp.sh:cached
+
+  alice-ident:
+    image: provide/ident
+    container_name: alice-ident
+    depends_on:
+      - alice-nats-streaming
+      - alice-postgres
+      - alice-redis
+    environment:
+      - DATABASE_HOST=alice-postgres
+      - DATABASE_NAME=ident_dev
+      - DATABASE_USER=ident
+      - DATABASE_PASSWORD=ident
+      - NATS_CLIENT_PREFIX=alice-ident
+      - NATS_URL=nats://alice-nats:4222
+      - NATS_STREAMING_URL=nats://alice-nats-streaming:4222
+      - REDIS_HOSTS=alice-redis:6379
+      - LOG_LEVEL=DEBUG
+      - PORT=8080
+    healthcheck:
+      test: ["CMD", "curl", "-f", "http://alice-ident:8080/status"]
+      interval: 1m
+      timeout: 1s
+      retries: 2
+      start_period: 10s
+    hostname: alice-ident
+    networks:
+      - alice
+    ports:
+      - 8081:8080
+    restart: always
+
+  alice-ident-consumer:
```

Comment:
What is purpose of `alice-ident-consumer` versus `alice-ident`?

Reply:
This is documented in the forthcoming README, but `alice-ident` exposes a REST API that implements Alice's local registry/workgroup interface. `alice-ident-consumer` handles async tasks in an idempotent/fault-tolerant way.